### ISSUE TITLE
Problem setting selection after setting display:none on parent element (Firefox only)

### ISSUE THREAD

**Issue by timdown** [timdown/rangy#19]
_From [mrchrisa...@gmail.com](https://code.google.com/u/112373757591860189964/) on November 20, 2010 15:18:50_

What steps will reproduce the problem? 1. Open attached test page in Firefox (I'm using 3.6.12)
2. Click the following buttons: populate, select, hide, show, select What is the expected output? What do you see instead? I expected the letter 'e' to be selected. This works OK on Chrome, but not Firefox (note the attached test case won't work on IE though, that's nothing to do with Rangy). What version of the product are you using? On what operating system? rangy-1.0b1 Please provide any additional information below. As a workaround, calling selection.detach() after making the element visible solves the problem.

**Attachment:** [test.html](http://code.google.com/p/rangy/issues/detail?id=19)

_Original issue: http://code.google.com/p/rangy/issues/detail?id=19_
